components/grid.css

```diff
@@ -0,0 +1,683 @@
+.container {
+  width: 100%;
+  max-width: 1200px;
```

Comment:
Can use clamp here, for width like: `width: clamp(min_width, 100%, 1200px)`

Reply:
okay, thanks for pointing it out, i will use clamp here, didn't even think of it.